### PR TITLE
feat(cli): add option to remove source file after decompression

### PR DESCRIPTION
**Pull request by ttys3**
add cli flag to allow auto remove source file after decompression, this is often the common usage

like https://github.com/aonez/Keka on Mac, simple double click and file extracted, source file removed.

### CODE REVIEW

File: src/cli/args.rs [hunk 1]
```diff
@@ -88,6 +88,10 @@ pub enum Subcommand {
         /// Place results in a directory other than the current one
         #[arg(short = 'd', long = "dir", value_hint = ValueHint::FilePath)]
         output_dir: Option<PathBuf>,
+
+        /// Remove the source file after successful decompression
+        #[arg(short = 'r', long, default_value_t = false)]
```
Comment (marcospb19) [inline src/cli/args.rs:93]: false is the default value for booleans right? :thinking:

**Review by marcospb19 — APPROVED** [review]
great! can you add an entry to CHANGELOG.md for this feature?

File: src/cli/args.rs (hunk 1 above)
Comment (ttys3) [reply src/cli/args.rs:93]: just want it to be more explicit. because remove file is a danger operation.

yes, I'm OK to remove that.

**Comment by ttys3** [comment]
> great! can you add an entry to CHANGELOG.md for this feature?

done

**Review by marcospb19 — APPROVED** [review]
> done

Thank you!

**Pull request merged**